src/current/cockroachcloud/cluster-management.md

```diff
@@ -68,53 +68,50 @@ To add or remove nodes from your cluster:
 
     The **Edit cluster** page displays.
 
-1. From the **Nodes** dropdown, select the number of nodes you want in each region.
-1. In the **Summary** sidebar, verify the hourly estimated cost for the cluster.
-1. Click **Next: Payment**.
-1. On the **Summary** page, verify your new cluster configuration.
-1. Click **Update**.
+1. On the **Regions** page, select the number of nodes you want in each region from the **Nodes** dropdown.
+1. In the sidebar, verify the hourly estimated cost for the cluster.
+1. Click **Next: Capacity** and then **Next Finalize**.
```

Comment:
Do you need to choose anything in Capacity before you go to the Finalize step?

Reply:
Nope!